flake.nix

```diff
@@ -68,19 +68,19 @@
          });
          cddl-tools = (final: prev: {
           cbor-diag = final.callPackage ./pkgs/cbor-diag { };
-          cddl = final.callPackage ./pkgs/cddl { };          
+          cddl = final.callPackage ./pkgs/cddl { };
          });
        };
        supportedSystems = [
             "x86_64-linux"
             "x86_64-darwin"
             "aarch64-linux"
             "aarch64-darwin"
-       ];       
+       ];
     in flake-utils.lib.eachSystem supportedSystems (system:
-         let 
+         let
            pkgs = import nixpkgs {
-             overlays = [haskellNix.overlay] ++ __attrValues overlays;
+             overlays = [haskellNix.overlay] ++ builtins.attrValues overlays;
```

Comment:
What's up with all of these __ -> builtins renaming? I honestly don't know which ones are better, __ was just shorter 🙄

Reply:
It's just that https://github.com/oxalica/nil complain that `__x` isn't defined :)

Reply:
fair. I'd argue that's a bug in `nil`, but alright, that's a reason I can accept.